site-src/geps/gep-1742.md

```diff
@@ -0,0 +1,315 @@
+# GEP-1742: Timeouts
+
+* Issue: [#1742](https://github.com/kubernetes-sigs/gateway-api/issues/1742)
+* Status: Provisional
+
+(See status definitions [here](overview.md#status).)
+
+## TLDR
+
+Create some sort of design so that Gateway API objects can be used to configure
+timeouts for different types of connection.
+
+## Goals
```

Comment:
This is where we have to decide whether or not we want to be opinionated about what a timeout is